src/input/label.component.ts

```diff
@@ -56,6 +56,7 @@ import { PasswordInput } from "./password.directive";
 					[warn]="warn"
 					[warnText]="warnText"
 					[ariaLabel]="ariaLabel"
+					[readonly]="readonly"
```

Comment:
Readonly value is being passed to the `cds-textarea-label` component instead of `cds-text-label`. Additionally, in `cds-text-label` component (see `text-input-label.component.ts`), the host should be assigned `cds--text-input-wrapper--readonly` class.

```ts
@Input() @HostBinding('class.cds--text-input-wrapper--readonly') readonlyWrapper = false;
```

With this class assigned, it should look like this:

![image](https://github.com/user-attachments/assets/641b906c-9779-4e0a-8f69-ae1970f489d5)

Reply:
Hey @Akshat55 , I have made the changes as per your suggestion.
Please let me know if I am missing anything :)

Reply:
Sorry for the delay, had to think about this to reduce setting readonly twice. The solution I came up with was to use a getter and apply the class based on `readonly` attribute of the input.